server/printing/SCEPrinter.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+#!/usr/bin/env python3
+import curses
+import argparse
+parser = argparse.ArgumentParser(description='Print the mock for 2D printer.')
+parser.add_argument('--mock-output', help="Displays Mock Output", )
+args = parser.parse_args()
+
+
+class SCEPrinter(data):
```

Comment:
since this class will mock the printer, consider renaming it `ScePrinterMock` or something of the sort